js/app.js

```diff
@@ -84,16 +124,22 @@ function addEventListeners() {
 
 function pasteEventListener() {
   linkInput.addEventListener("paste", (event) => {
-    let url = event.clipboardData.getData("text");
+    let urlFull = event.clipboardData.getData("text");
```

Comment:
As the value of urlFull is never changed in this scope it can be declared as a **const** instead of a **let**